waterloop-site/src/theme/global.scss

```diff
@@ -44,6 +44,17 @@ h1 {
 }
 
 @media (max-width: 420px) {
+  .GooseRoster-Img {
+    /* type img */
+     /* type img */
+  display: block;
+  object-fit: cover; /*Centers image and crops instead of scales aspect ratio.*/
+  border: 10px;
+  max-width: 355px;
+  /*200 IQ bug fix (the first image doesn't load in correct size if only using width ;) */
```

Comment:
Wait that's the comment I originally left on the SimpleFeatures CSS component. Why is it still here? :o